readthedocs/oauth/models.py

```diff
@@ -90,14 +91,7 @@ class RemoteRepository(models.Model):
         User,
         verbose_name=_('Users'),
         related_name='oauth_repositories',
-    )
-    account = models.ForeignKey(
```

Comment:
We should probably add the `remote_id` field here as well and populate it with the `id` that it's in `remoterepository.json`.

If we are not doing it in this PR, we should create an issue to keep track of. We will need it to remove the usage of `full_name` in many places.

I created a project at https://github.com/orgs/readthedocs/projects/74?add_cards_query=is%3Aopen where we can track all of the related PR and issues. Let me know if you have permissions.

Reply:
Okay sure, maybe we can do data migration for this and https://github.com/readthedocs/readthedocs.org/pull/7536#discussion_r501637990 first. 

This PR will remove the json field from ``RemoteRepository`` model so I think its better to do this first in a different PR or make it the first migration on this PR.
@humitos What do you say?

Reply:
> I created a project at https://github.com/orgs/readthedocs/projects/74?add_cards_query=is%3Aopen where we can track all of the related PR and issues. Let me know if you have permissions.

@humitos I can not see some of the cards maybe they are from a private repo or something, otherwise it all good :+1: